saleor/payment/gateways/stripe/tests/test_plugin.py

```diff
@@ -214,6 +214,62 @@ def test_process_payment(
     assert not mocked_customer.called
 
 
+@patch("saleor.payment.gateways.stripe.stripe_api.stripe.Customer.create")
+@patch("saleor.payment.gateways.stripe.stripe_api.stripe.PaymentIntent.create")
+def test_process_payment_with_disabled_include_receipt_email(
```

Comment:
Lack of `given`, `when`, `then`, and also we should also add a test when `include_receipt_email` is `True`